Functions.Rmd

```diff
@@ -1230,7 +1230,7 @@ second(x) <- 5L
 x
 ```
 
-I say they "act" like they modify their arguments in place, because, as discussed in [Modify-in-place], they actually create a modified copy. We can see that by using `tracemem()`:
+I say they act like they modify their arguments in place, because, as explained in Section \@ref(modify-in-place) about modify-in-place behaviour, they actually create a modified copy. We can see that by using `tracemem()`:
```

Comment:
```suggestion
I say they act like they modify their arguments in place, because, as explained in Section \@ref(modify-in-place), they actually create a modified copy. We can see that by using `tracemem()`:
```